index.html

```diff
@@ -38,5 +38,118 @@ <h2 class="text-container" id="sdsection">
       <img src="arrow-button.svg" class="arrow-button"><a href="#"></a></img>
       </div>
     </section>
+    <section class="works">
```

Comment:
- [ ] All your cards in this section render the same content. Kindly add the right content for each card from the Figma design. Please note that each card has a different heading and a different image.

Reply:
Hi, I don't find those differences. Could yo provide me some examples?